public/app/features/dashboard-scene/settings/variables/components/ConstantVariableForm.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FormEvent } from 'react';
+
+import { selectors } from '@grafana/e2e-selectors';
+
+import { VariableLegend } from './VariableLegend';
+import { VariableTextField } from './VariableTextField';
+
+interface ConstantVariableFormProps {
+  constantValue: string;
+  onChange: (event: FormEvent<HTMLInputElement>) => void;
+  onBlur: (event: FormEvent<HTMLInputElement>) => void;
+}
+
+export function ConstantVariableForm({ onChange, onBlur, constantValue }: ConstantVariableFormProps) {
+  return (
+    <>
+      <VariableLegend>Constant options</VariableLegend>
+      <VariableTextField
+        value={constantValue}
+        name="Value"
+        placeholder="your metric prefix"
+        onChange={onChange}
+        onBlur={onBlur}
+        testId={selectors.pages.Dashboard.Settings.Variables.Edit.ConstantVariable.constantOptionsQueryInputV2}
+        width={30}
+      />
```

Comment:
In the `CustomVariableForm`, I got some feedback that we could remove the need to use temporal state, since we can just stop passing vale to the inputs, and use the `defaultValue` prop. 
```suggestion
      <VariableTextField
        defaultValue={constantValue}
        name="Value"
        placeholder="your metric prefix"
        onBlur={onChange}
        testId={selectors.pages.Dashboard.Settings.Variables.Edit.ConstantVariable.constantOptionsQueryInputV2}
        width={30}
      />
```
And the `ConstantVariableForm` will call `onChange` when `onBlur`, having a consistent experience with all one as well